### ISSUE TITLE
Undefined variable flagged in consecutive lines

### ISSUE THREAD

**Issue by paulschreiber** [sirbrillig/phpcs-variable-analysis#203]
I would expect to only see one error here:
```php
16 <?php if ( $error ) : ?>
17 <code><?php echo esc_html( $error ); ?></code>
18 <?php endif; ?>
```

It's possible this is intentional. I wanted to flag it in case it was not. (This is a WordPress `template-part`; variables are defined in the parent.)

```
FILE: template-parts/mail-error.php
-----------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
-----------------------------------------------------------------------------------------------------------------
 16 | WARNING | Variable $error is undefined.
    |         | (VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable)
 17 | WARNING | Variable $error is undefined.
    |         | (VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable)
-----------------------------------------------------------------------------------------------------------------
```

**Comment by sirbrillig** [comment]
It is intentional (they are both uses of an undefined variable). However, if you plan on scanning a lot of template files, you might want to ignore undefined variables in the file scope using the [allowUndefinedVariablesInFileScope](https://github.com/sirbrillig/phpcs-variable-analysis/pull/193) option, added in v3.0.0-beta.4.

**Comment by sirbrillig** [comment]
Probably worth mentioning that 3.0 is just pending at least a beta release of [phpcsutils](https://phpcsutils.com/).
